src/helpers/copy-to-clipboard.js

```diff
@@ -1,3 +1,9 @@
-export default function copyToClipboard(content) {
-    //ma kopiować do schowka
+export default function copyToClipboard() {
+    navigator.clipboard.writeText(this.code)
```

Comment:
Nie istnieje w tym pliku this.code. Przywróć parametr content i przekaz tam tekst do skopiowania. Nie naruszaj proszę narzuconej struktury.

Reply:
Poprawiłem.